content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -4,185 +4,229 @@ Process receipts and deliveries in two steps
+   warehouse locations in the database, named `WH/Input` and `WH/Output`, respectively.
 
-Process a receipt in two steps (input + stock)
-==============================================
+   To rename or edit these locations, navigate to :menuselection:`Inventory app --> Configuration
+   --> Locations`, and select the desired location.
 
-Create a purchase order
------------------------
+   On the location's form, change the :guilabel:`Location Name`, and make any other necessary
+   changes.
 
-On the main :menuselection:`Purchase` application dashboard, start by making a new quote by clicking
-:guilabel:`New`. Then, select (or create) a :guilabel:`Vendor` from the drop-down field, add a
-storable :guilabel:`Product` to the order lines, and click :guilabel:`Confirm Order` to finalize the
-quote as a new purchase order.
+Process receipt in two steps (input + stock)
+============================================
 
-.. important::
-   For businesses with multiple warehouses that have different step configurations, the
-   :guilabel:`Deliver To` field on the :abbr:`PO (Purchase Order)` form may need to be specified as
-   the correct *input location* connected to the two-step warehouse, which can be done by selecting
-   the warehouse from the drop-down selection that includes the `Receipts` label at the end of the
-   name.
+When products are received in two steps, they move from the vendor location to an input location,
+then from the input location to warehouse stock in the database upon validation of a purchase order
+(PO) and a subsequent internal transfer.
+
+Create purchase order
+---------------------
 
-After confirming the :abbr:`PO (Purchase Order)`, a :guilabel:`Receipt` smart button appears at the
-top of the :abbr:`PO (Purchase Order)` form. Click it to reveal the new receipt (WH/IN) operation.
+To create a |PO|, navigate to the :menuselection:`Purchase app`, and click :guilabel:`New`. This
+opens a blank :guilabel:`Request for Quotation` (RfQ) form.
+
+Add a vendor in the :guilabel:`Supplier` field. Then, fill out the various fields on the |RfQ|, as
+necessary.
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-new-rfq.png
+   :align: center
+   :alt: Filled out new Request for Quotation from vendor.
+
+Under the :guilabel:`Products` tab, click :guilabel:`Add a product`, and select a product to add to
+the |RfQ|.
+
+Once ready, click :guilabel:`Confirm Order`. This moves the |RfQ| to the :guilabel:`Purchase Order`
+stage.
+
+Once the |PO| is confirmed, a :guilabel:`Receipt` smart button appears at the top of the form.
+Clicking the smart button opens the warehouse receipt (WH/IN) form.
 
 .. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-smart-button.png
    :align: center
-   :alt: After confirming a purchase order, a Receipt smart button will appear.
+   :alt: Delivery smart button for validated purchase order.
```

Comment:
Why does it only say 1 receipt, and we have two steps enabled? may be good to say that the internal transfer is created later

Reply:
In 17.2 this is how it appears and works as intended per the change in routes functions